docs/docs/deploy/digital-ocean-app-platform.md

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_position: 3
+---
+
+# Digital Ocean App Platform 🌊
+
+[App Platform](https://www.digitalocean.com/products/app-platform) is a service from [Digital Ocean](https://www.digitalocean.com/) that helps you launch apps quickly while they manage the underlying infrastructure.
+
+## Prerequisites
+
+Before you get started, if you haven't already completed these steps, you'll have to:
+
+1. Create a free [Digital Ocean account](https://cloud.digitalocean.com/registrations/new) (or sign in with Google or Github).
+
+:::caution
+You can incur costs when running this quickstart! For more details, see [App Platform Pricing](https://www.digitalocean.com/pricing/app-platform) and [Container Registry Pricing](https://www.digitalocean.com/pricing/container-registry).
+:::
+
+2. Install [Docker](https://docs.docker.com/get-docker/) on your machine, and you'll have to start the app. You can verify it is set up correctly by running:
+
+```bash
+docker images
+```
+
+(If Docker is running, the command will print the images on your machine. If not, it will print something like `Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?`)
+
+3. Install the [Digital Ocean CLI (doctl)](https://docs.digitalocean.com/reference/doctl/how-to/install/) on your machine.
+
+4. [Create a Digital Ocean API Token](https://docs.digitalocean.com/reference/api/create-personal-access-token/) for your account with read and write access, then use it to give `doctl` access to your account by running (it will prompt you for the access token):
```

Comment:
```suggestion
4. [Create a Digital Ocean API Token](https://docs.digitalocean.com/reference/api/create-personal-access-token) for your account with read and write access, then use it to give `doctl` access to your account by running (it will prompt you for the access token):
```